packages/webapp/pages/onboarding.tsx

```diff
@@ -124,6 +113,8 @@ export function OnboardPage(): ReactElement {
   const targetId: string = ExperimentWinner.OnboardingV4;
   const formRef = useRef<HTMLFormElement>();
   const [activeScreen, setActiveScreen] = useState(OnboardingStep.Intro);
+  const shouldEnrollContentType = useRef(false);
```

Comment:
The name is a bit confusing here as this state has nothing to do with enrolment.
It should be something like `shouldShowContentTypeScreen` maybe?

Reply:
renamed to enableContentTypeStep